api/v1/views/__init__.py

```diff
@@ -0,0 +1,10 @@
+#!/usr/bin/python3
+"""
+Initial Blueprint
+"""
+
+
+from flask import Flask, Blueprint
```

Comment:
why import Flask???

Reply:
that was an error, I'm going to change that and fix it